app.py

```diff
@@ -0,0 +1,31 @@
+import os
+from flask import Flask, jsonify, request
+
+app = Flask(__name__)
+
+currid = [{ 'currid': 1 }]
+
+@app.route('/')
+def hello():
+    return 'Hello World!'
+
+@app.route('/todo/api/v1.0/currid', methods=['GET'])
+def get_id():
+	return jsonify({'currid': currid})
+
+@app.route('/todo/api/v1.0/currid', methods=['POST'])
+def post_id():
+	if not request.json:
+		print "could not find request.json"
+	if not 'newid' in request.json:
+		print "double fuck me like a rack of boar"
```

Comment:
i noticed that this line you're printing isn't very informative. generally it's good to include information in your print statements. for example if `newid` is not found in `request.json` you might want to know what is in `request.json`.

also tone down your fucking language. unbelievable. :-1:  :-1:  :-1:  :-1:  :-1:  :-1: :-1: :-1: :-1:  :-1: :-1:  :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: :-1: